_posts/en/newsletters/2018-07-24-newsletter.md

```diff
@@ -0,0 +1,169 @@
+---
+title: 'Bitcoin Optech Newsletter #5'
+permalink: /en/newsletters/2018/07/24/
+name: 2018-07-24-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+version: 1
+---
+This week's newsletter includes news about a new method under
+development for describing which outputs you'd like to find, the merge
+of partially-signed Bitcoin transactions into Bitcoin Core, and several
+other notable Bitcoin Core merges.
+
+## Action items
+
+- Bitcoin Core [0.16.2RC2][] released for testing in preparation for a
+  maintenance release that will provide bugfixes and backports.
+  Community testing is highly appreciated.  Note, there was no RC1 due
+  to a metadata problem being detected and corrected after RC1 was
+  tagged but before it was released.
+
+## Dashboard items
+
+- Transaction fees are lower than they were this time last week.  Anyone
+  who can wait 12 or more blocks for confirmation can reasonably pay the
+  default minimum feerate.  It's a good time to [consolidate UTXOs][].
+
+- The number of [native segwit outputs][p2shinfo bech32] had been
+  increasing steadily over time, but dropped by about 400,000 (80%) this
+  week, probably as part of a consolidation by an exchange.  The average
+  number of new native segwit outputs created per hour remains
+  relatively constant, indicating no obvious decrease in adoption.
+  (Remember that if your service accepts payments to native segwit
```

Comment:
I've reworded this to (I think) make it a bit clearer. Thoughts?

Reply:
I didn't really like my parenthetical version, and I still don't really like the non-parenthetical version, so I'm just going to drop the whole thing.  That segwit saves money is well known and I'm sure we'll have better opportunities in the future to mention the amount of savings possible, so it doesn't seem important to include it here.

Reply:
That's a shame. I like reminding companies of the benefits of segwit whenever I can!

But you're right, this doesn't need to be here. There will be better opportunities in future.